bin/deploy-wmagent.sh

```diff
@@ -230,9 +230,11 @@ cd -
 fi
 echo "Done!" && echo
 
-### Enabling couch watchdog:
+### Enabling couch watchdog; couchdb fix for file descriptors
 echo "*** Enabling couch watchdog ***"
 sed -i "s+RESPAWN_TIMEOUT=0+RESPAWN_TIMEOUT=5+" $CURRENT/sw*/$WMA_ARCH/external/couchdb*/*/bin/couchdb
+sed -i "s+exec 1>&-+exec 1>$CURRENT/install/couchdb/logs/stdout.log+" $CURRENT/sw*/$WMA_ARCH/external/couchdb*/*/bin/couchdb
+sed -i "s+exec 2>&-+exec 2>$CURRENT/install/couchdb/logs/stderr.log+" $CURRENT/sw*/$WMA_ARCH/external/couchdb*/*/bin/couchdb
```

Comment:
And since that change was reverted back, I think I can just keep it as it is now :-)